server/src/main/java/com/arcadedb/server/ha/HAServer.java

```diff
@@ -88,7 +102,18 @@ public class HAServer implements ServerPlugin {
   private             Thread                                         electionThread;
 
   public enum QUORUM {
-    NONE, ONE, TWO, THREE, MAJORITY, ALL
+    NONE, ONE, TWO, THREE, MAJORITY, ALL;
+
+    public int quorum(int numberOfServers) {
+      return switch (this) {
+        case NONE -> 0;
+        case ONE -> 1;
+        case TWO -> 2;
+        case THREE -> 3;
+        case MAJORITY -> numberOfServers / 2 + 1;
+        case ALL -> numberOfServers;
+      };
```

Comment:
Shouldn't we have the default here that throws an exception like with the original code?

Reply:
No, this is the "trick" of using the Enum itself. You can only call the method if you are using one of the Enuma: QUORUM. ONE/TWO/THREE/MAJORITY/ALL.

The parsing of the value from the conf is in ReplicatedDatabase line 100. 

No more need for exception

Reply:
@lvca  may you approve this?